app/models/manageiq/providers/hawkular/datawarehouse_manager.rb

```diff
@@ -75,5 +83,15 @@ def self.ems_type
     def self.description
       @description ||= "Hawkular Datawarehouse".freeze
     end
+
+    def self.event_monitor_class
+      ManageIQ::Providers::Hawkular::DatawarehouseManager::EventCatcher
+    end
+
+    # To blacklist defined event types by default add them here...
```

Comment:
@moolitayer if this is an issue now and we can't dedicate time to properly implement it, take it out of the PR.